frontend/src/component/pages/Home.tsx

```diff
@@ -206,12 +210,28 @@ export class Home extends Component<Props, State> {
     this.props.store.dispatch(raiseInputError(err));
   };
 
+  private copyShortenedLink = (shortenedLink: string) => {
```

Comment:
Let's put a TODO here so that we can refactor this into `ShortLinkService` in the future to reduce the complexity of Home page. Here is the TODO format: TODO(issue#): refactor into `ShortLinkService` to decouple business logic from view.